### PR TITLE
Bug 1159438 - add support for ``parse_status`` of logs

### PR DESCRIPTION
**Pull request by camd**
This is to support the new field made available in [Bug 1080760](https://bugzilla.mozilla.org/show_bug.cgi?id=1080760)

### CODE REVIEW

File: thclient/client.py [hunk 1]
```diff
@@ -210,10 +210,13 @@ def add_option_collection(self, option_collection):
         if option_collection:
             self.data['job']['option_collection'].update(option_collection)
 
-    def add_log_reference(self, name, url):
+    def add_log_reference(self, name, url, parse_status='pending'):
+        """
+        parse_status - one of 'pending', 'parsed' or 'error'
```
Comment (maurodoglio) [inline thclient/client.py]: can you check the parse status is one of the accepted values?

Comment (camd) [inline thclient/client.py]: yep, good call.  Done

**Comment by maurodoglio** [comment]
@camd can you rebase on top of what's on master please?

**Comment by camd** [comment]
@maurodoglio Yep.  rebased on top of the pep8 fixes from wlach

**Pull request merged**